webpack.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = merge(baseConfig, extractCssConfig, {
     rules: [
       { parser: { system: false } },
       {
-        test: /fonts\/.+\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,
+        test: /fonts(\\|\/).+\.(woff(2)?|ttf|eot|svg)(\?v=\d+\.\d+\.\d+)?$/,
```

Comment:
I think cross-env could solve also this issue

Reply:
We tried without success :'(